### ISSUE TITLE
Agree a whitespace convention and stick to it

### ISSUE THREAD

**Issue by davidcmoulton** [elifesciences/elife-bot#159]
Some files have mixed tabs and spaces (e.g. `decider.py`). Whitespace inconsistency in Python can cause problems, should be fixed.

See PEP8:

- "Spaces are the preferred indentation method...Python 2 code indented with a mixture of tabs and spaces should be converted to using spaces exclusively" (https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces).
- "Use 4 spaces per indentation level." (https://www.python.org/dev/peps/pep-0008/#indentation)

To keep a clean diff, commits addressing this whitespace issue should not contain any other types of change.

**Comment by jhroot** [comment]
I'd suggest merging EXP back into master before doing this, or do it in both, or it could make the merge much harder. 

Would suggest using PEP-8 (and new files have been created following those guidelines)

**Comment by davidcmoulton** [comment]
Agreed: it makes sense to wait until after EXP has been merged into master.
